tests/basictests/jupyterhub/jhtest.py

```diff
@@ -42,14 +43,15 @@ def load_config(self):
         self.url = os.environ.get('JH_URL')
         self.username = os.environ.get('JH_LOGIN_USER')
         self.password = os.environ.get('JH_LOGIN_PASS')
+        self.login_provider = os.environ.get('OPENSHIFT_LOGIN_PROVIDER', 'htpasswd-provider')
         self.notebooks = os.environ.get('JH_NOTEBOOKS', "").split(",")
         self.user_name = os.environ.get('JH_USER_NAME', 'test-user1')
         self.spawner = {
             "image": os.environ.get('JH_NOTEBOOK_IMAGE', "s2i-minimal-notebook:3.6"),
             "size": os.environ.get('JH_NOTEBOOK_SIZE', "None"),
         }
-        self.as_admin = os.environ.get('JH_AS_ADMIN', False)
-        self.headless = os.environ.get('JH_HEADLESS', False)
+        self.as_admin = strtobool(os.environ.get('JH_AS_ADMIN', False))
+        self.headless = strtobool(os.environ.get('JH_HEADLESS', False))
         self.preload_repos = os.environ.get('JH_PRELOAD_REPOS', "https://github.com/vpavlin/jh-stresstest")
```

Comment:
Unrelated to this PR, but is jh-stresstest anything we'd want to house under opendatahub-io, or is it perfectly happy living in your github?

Reply:
The purpose of the repo here is to provide a test notebook (https://github.com/vpavlin/jh-stresstest/blob/master/test.ipynb). We can definitely move the notebook and enhance it. No objections there. Do you want me to do it in this PR?

Reply:
Nah, no need to do it in this PR.